src/components/Time.jsx

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
Remove empty lines.

Reply:
I have removed those lines in [commit 0c7598d](https://github.com/SolidLabResearch/generic-data-viewer/pull/17/commits/0c7598de586674e3f7ec4a020a19d1115c1df3cf)